src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.VariableIdentifier.cs

```diff
@@ -49,7 +48,7 @@ public override bool Equals(object obj)
                 return !left.Equals(right);
             }
 
-            internal string GetDebuggerDisplay()
```

Comment:
>GetDebuggerDisplay [](start = 28, length = 18)

I'm curious, why the change?
Just want to confirm it's not because `DebuggerDisplay` isn't working in dev16 (I've already filled a bug on that).

Reply:
Changed because `ToString()` is used in the EE for instances in a collection.